ngraph_bridge/ngraph_builder.cc

```diff
@@ -1809,9 +1809,8 @@ static Status TranslateFusedBatchNormOp(
   shared_ptr<ng::Node> ng_input, ng_scale, ng_offset, ng_mean, ng_variance;
   bool is_v3 = op->type_string() == "FusedBatchNormV3";
   if (tf_is_training) {
-    TF_RETURN_IF_ERROR(GetInputNode(ng_op_map, op, 0, &ng_input));
-    TF_RETURN_IF_ERROR(GetInputNode(ng_op_map, op, 1, &ng_scale));
-    TF_RETURN_IF_ERROR(GetInputNode(ng_op_map, op, 2, &ng_offset));
+    NGRAPH_VLOG(0) << "FusedBatchNorm - training is not supported";
+    return Status::OK();
```

Comment:
Do not mark `FusedBatchNorm` for clustering if the `is_training` attribute is present.